tests/Service/ProgrammesService/CountAllTleosByCategoryTest.php

```diff
@@ -2,21 +2,27 @@
 
 namespace Tests\BBC\ProgrammesPagesService\Service\ProgrammesService;
 
+use BBC\ProgrammesPagesService\Domain\Entity\Genre;
+
 class CountAllTleosByCategoryTest extends AbstractProgrammesServiceTest
 {
-    public function testCountAllTleosByCategory()
+    public function testProtocolWithRepository()
```

Comment:
Not quite sure what this test name means, can you make it clearer?

Reply:
Yes of course! jeje